release-notes/release-notes59.md

```diff
@@ -215,6 +215,22 @@ The {{site.productname}} 5.9 release includes an accompanying release of the **P
 
 For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseurl}}/plugins/premium/powerpaste/).
 
+### Real-Time Collaboration 1.0.1
+
+The {{site.productname}} 5.9 release includes the first generally available release of the **Real-Time Collaboration (RTC)** premium plugin (version 1.0.0).
```

Comment:
Even though 1.0.1 is the build in the release, I think the heading should remain 1.0.0 as the text says - or at the very least they should be the same.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/2091/commits/6a626cb3c875b57a5f260de41ee39ae7179b61ad